From a01d276475b8c095468568770ee67380f266361a Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@nxp.com>
Date: Wed, 5 Sep 2018 21:04:43 +0800
Subject: [PATCH 4576/5242] MLK-18835 ARM: dts: imx7ulp-evk: add off-on-delay
 for usdhc

commit  4f56b00cec9ad85bc5642589aadf50303d2da918 from
https://source.codeaurora.org/external/imx/linux-imx.git

Make sure the time between power off and power on sd card meet the
sd SPEC, otherwise after suspend and resume, SD3.0 card will no longer
be recognized as SD3.0, but SD2.0

Signed-off-by: Haibo Chen <haibo.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-evk-sd1.dts |    1 +
 arch/arm/boot/dts/imx7ulp-evk.dts     |    1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/imx7ulp-evk-sd1.dts b/arch/arm/boot/dts/imx7ulp-evk-sd1.dts
index 154b786..e5bdf04 100644
--- a/arch/arm/boot/dts/imx7ulp-evk-sd1.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk-sd1.dts
@@ -18,6 +18,7 @@
 			regulator-min-microvolt = <3300000>;
 			regulator-max-microvolt = <3300000>;
 			gpio = <&gpio_pte 11 GPIO_ACTIVE_HIGH>;
+			off-on-delay = <20000>;
 			enable-active-high;
 		};
 	};
diff --git a/arch/arm/boot/dts/imx7ulp-evk.dts b/arch/arm/boot/dts/imx7ulp-evk.dts
index d7abcfb..b939710 100644
--- a/arch/arm/boot/dts/imx7ulp-evk.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk.dts
@@ -77,6 +77,7 @@
 			regulator-min-microvolt = <3300000>;
 			regulator-max-microvolt = <3300000>;
 			gpio = <&gpio_ptd 0 GPIO_ACTIVE_HIGH>;
+			off-on-delay = <20000>;
 			enable-active-high;
 		};
 
-- 
1.7.9.5

